.travis.yml

```diff
@@ -56,8 +56,11 @@ addons:
     sources:
       # sources list: https://github.com/travis-ci/apt-source-safelist/blob/master/ubuntu.json
       - sourceline: 'ppa:qbittorrent-team/qbt-libtorrent-travisci'
+      - sourceline: 'deb https://apt.kitware.com/ubuntu/ bionic main'
+        key_url: 'https://apt.kitware.com/keys/kitware-archive-latest.asc'
     packages:
       # packages list: https://github.com/travis-ci/apt-package-safelist/blob/master/ubuntu-trusty
+      - cmake
       - [autoconf, automake, colormake]
```

Comment:
I would prefer to keep it sorted: `[autoconf, automake, cmake, colormake]`

Reply:
@Chocobo1
Fixed.